### PR TITLE
Exclude sdk/php pages from index

### PR DESCRIPTION
**Pull request by obaybakova**
# Pull request motivation(s)

We need to exclude pages under /sdk/php from the search results.

### What is the current behaviour?

When we search for "php sdk", we see pages in the section in the search results.

### What is the expected behaviour?

When we search for "php sdk", we find nothing.

### CODE REVIEW

File: configs/xsolla.json [hunk 1]
```diff
@@ -82,6 +82,7 @@
   "stop_urls": [
     "/search",
     "/changelog/tags",
+    "https://developers.xsolla.com(/[a-z][a-z])?/sdk/php/*",
```
Comment (shortcuts) [inline configs/xsolla.json:85]:
```suggestion
    "/sdk/php/",
```

If the goal is to remove any pages under the `/sdk/php/`, this will do it!

Comment (obaybakova) [reply configs/xsolla.json:85]: We also have a locale identifier right after the domain like 
https://developers.xsolla.com/de/sdk/php/
https://developers.xsolla.com/ja/sdk/php/
https://developers.xsolla.com/ko/sdk/php/

Should it be `(/[a-z][a-z])?/sdk/php/` then?

Comment (shortcuts) [reply configs/xsolla.json:85]: `stop_urls` checks matching patterns, meaning as long as `/sdk/php/` is in the URL, it will be ignored

Comment (obaybakova) [reply configs/xsolla.json]: Got it, thanks!

**Pull request merged**